### ISSUE TITLE
Problem with long text

### ISSUE THREAD

**Issue by ziadhassan7** [sergivonavi/MaterialBanner#10]
This library is great but when I use long text, it doesn't show up probably, some of the text are hidden!

**Comment by alexatbooli** [comment]
Yeah there seems to be a timing bug with setting widths - sometimes the text wraps properly, sometimes some words are lost off the right hand edge of the screen, hidden behind the margin/padding.
